GitWorkflow/GitWorkflow/main.cpp

```diff
@@ -1,37 +1,27 @@
 #include <vector>
 #include <chrono>
 
-#include <SFML/Graphics.hpp>
+#include "Platform.hpp"
 #include "Player.hpp"
-#include "Utility.hpp"
+#include "Level.hpp"
```

Comment:
Platform.hpp and Player.hpp are included via Level.hpp. Remove those includes here